src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -523,11 +523,11 @@ private String getAbsoluteName(ObjectType objectType) {
     }
 
     private String getUnqualifiedName(TypedVar symbol) {
-      return lastPart(symbol.getName());
+      return getUnqualifiedName(symbol.getName());
     }
 
-    private String getUnqualifiedName(JSType type) {
```

Comment:
No, I'm just unifying all calls to end up on the same call path, so that you cannot e.g. change `getUnqualifiedName` below, and then one method was unused - this change is mostly unrelated.

The actual problem was as mentioned above that the enum that was aliased was in a higher package than the lower enum, which due to the lookup mechanism of TypeScript caused some aliasing snafu.